lib/ii.cpp

```diff
@@ -12418,6 +12418,9 @@ grn_ii_similar_search_internal(grn_ctx *ctx, grn_ii_select_data *data)
   grn_rc rc = GRN_SUCCESS;
   grn_hash *h;
   grn_token_cursor *token_cursor;
+  if (data->query_len == 0) {
+    return GRN_SUCCESS;
```

Comment:
直前で定義されている`rc`を利用するのでも良さそうかなと思いました。
```suggestion
    return rc;
```

Reply:
`GRN_SUCCESS`を使うことで、`data->query_len == 0`（空文字列が入ってきた）時は何もしないということが伝わりやすいかと思うので、このままにしようと思います。